### ISSUE TITLE
Grid stencil generated indices not ordered

### ISSUE THREAD

**Issue by fredrikbk** [simit-lang/simit#33]
According to e95fae8557a3 the CSR sinks of indices assembled from stencils are not ordered. If this is true then element wise operations on these matrices will not work, since the merge operations rely on ordering.

**Comment by fredrikbk** [comment]
@gkanwar, is this still the case?

**Comment by gkanwar** [comment]
Yes, this is still an issue. I can address this when I get back into town tomorrow.
